book/webpack-encore.rst

```diff
@@ -0,0 +1,148 @@
+Using Webpack Encore for your website assets
```

Comment:
For me this should be more in the `cookbook` instead of `book`.

Reply:
👍